packages/p/pam/package.yml

```diff
@@ -1,6 +1,6 @@
 name       : pam
 version    : 1.5.3
-release    : 28
+release    : 29
```

Comment:
You need to build PAM and make sure the pspec is included in the commit. It looks like you just bumped it without building it?